module/secrets.tf

```diff
@@ -1,13 +1,26 @@
-resource "aws_secretsmanager_secret" "client_key" {
-  count       = var.config.use_secrets_manager ? 1 : 0
-  name        = "/${local.name}/mesh/MESH_CLIENT_KEY"
-  description = "/${local.name}/mesh/MESH_CLIENT_KEY"
-  kms_key_id  = aws_kms_key.mesh.key_id
-}
-
-resource "aws_secretsmanager_secret" "shared_key" {
-  count       = var.config.use_secrets_manager ? 1 : 0
-  name        = "/${local.name}/mesh/MESH_SHARED_KEY"
-  description = "/${local.name}/mesh/MESH_SHARED_KEY"
-  kms_key_id  = aws_kms_key.mesh.key_id
-}
\ No newline at end of file
+
+moved {
```

Comment:
is that new?

Reply:
not especially 
![image](https://github.com/NHSDigital/terraform-aws-mesh-client/assets/449892/c377e338-c48b-46e5-bd4b-44cdef8d8146)